lib/utils/debugFilterImages.js

```
@@ -0,0 +1,54 @@
+const { createClient } = require("@supabase/supabase-js");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)